curweather/curweather.php

```diff
@@ -25,42 +25,46 @@ function getWeather( $loc, $units='metric', $lang='en', $appid='', $cachetime=0)
     $cached = Cache::get('curweather'.md5($url));
     $now = new DateTime();
     if (!is_null($cached)) {
-	$cdate = PConfig::get(local_user(), 'curweather', 'last');
-	$cached = unserialize($cached);
-	if ($cdate + $cachetime > $now->getTimestamp()) {
-	    return $cached;
-	}
+        $cdate = PConfig::get(local_user(), 'curweather', 'last');
```

Comment:
Can you please [use tabs](https://github.com/friendica/friendica/blob/develop/doc/Developers-Intro.md#coding-standards) for indentation and not spaces.

Reply:
The whole file uses spaces, and it wasn't supposed to be a complete formatting do-over. In this case I'd rather have the same indentation file-wise even if it's spaces.

Reply:
@tobiasd I will next time. Shall I change that file to tabs or leave it alone?

Reply:
Please change indentation to tabs, the goal is to have a unified code base with the same standards everywhere.